coderd/database/dbauthz/dbauthz.go

```diff
@@ -228,7 +228,7 @@ var (
 					rbac.ResourceOrgRoleAssignment.Type:  {rbac.ActionCreate},
 					rbac.ResourceUser.Type:               {rbac.ActionCreate, rbac.ActionUpdate, rbac.ActionDelete},
 					rbac.ResourceUserData.Type:           {rbac.ActionCreate, rbac.ActionUpdate},
-					rbac.ResourceWorkspace.Type:          {rbac.ActionUpdate},
+					rbac.ResourceWorkspace.Type:          {rbac.ActionUpdate, rbac.ActionDelete},
```

Comment:
Why does the system subject need to delete workspaces?

Reply:
I only needed it for my test unfortunately